app/models/thredded/topic.rb

```diff
@@ -44,7 +44,7 @@ class Topic < ActiveRecord::Base
                inverse_of:    :topics,
                counter_cache: :topics_count
 
-    has_many :posts,
+    has_many :posts, -> { order_oldest_first },
```

Comment:
Is this actually used anywhere? I thought all the current code establishes the order explicitly. Default scopes lead to some very unexpected behaviour. E.g. topic.posts.order_newest_first will no longer work (because it uses `order` which adds order conditions to the end).

> but it seemed a bit repititive to add the order to every single place we are using .posts...
There isn't a whole lot of these. We also use different ones (e.g. on the user's recent posts it's newest first, on the other views it's oldest first, elsewhere in the codebase outside of the view it's either one or something else entirely).